lib/init.coffee

```diff
@@ -103,7 +103,41 @@ module.exports =
         reResult = re.exec(toParse)
       ret
 
+    # only elixir 1.3+
     parseWarning = (toParse, textEditor) ->
+      console.log('to parse', toParse)
+
+      ret = []
+      re = ///
+        warning:\ (.*)\n # warning
+        \ \ (.*):([0-9]+) # file and file number
+        ///g
+      reResult = re.exec(toParse)
+
+      while reResult?
+        console.log(reResult)
+        try
+          ret.push
+            type: "Warning"
+            text: reResult[1]
+            filePath: path.join(projectPath(textEditor), reResult[2])
+            # use range, this because the previous method of getting a range
```

Comment:
Instead of completely abandoning the nice ranges provided by `rangeFromLineNumber`, how about we parse through [`atom.workspace.getTextEditors()`](https://atom.io/docs/api/v1.10.2/Workspace#instance-getTextEditors), checking the path they provide and send the proper editor to that if one is available?

If one isn't available, then a range like this is fine as it will be updated when that file is worked on, however the range should be a character wide, otherwise no highlight will be shown, and the cursor will be hidden if it is clicked on:

```coffeescript
new Range(
  [reResult[3] - 1, 0],
  [reResult[3] - 1, 1]
)
```